src/Http/WebUtilities/src/QueryHelpers.cs

```diff
@@ -113,13 +113,13 @@ public static string AddQueryString(string uri, IEnumerable<KeyValuePair<string,
         }
 
         var anchorIndex = uri.IndexOf('#');
-        var uriToBeAppended = uri;
-        var anchorText = "";
+        ReadOnlySpan<char> uriToBeAppended = uri;
+        ReadOnlySpan<char> anchorText = ReadOnlySpan<char>.Empty;
```

Comment:
```suggestion
        var uriToBeAppended = uri.AsSpan();
        var anchorText = ReadOnlySpan<char>.Empty;
```
To use more `var` here 😉